llm/server.go

```diff
@@ -798,7 +798,8 @@ func (s *llmServer) Completion(ctx context.Context, req CompletionRequest, fn fu
 
 	res, err := http.DefaultClient.Do(serverReq)
 	if err != nil {
-		return fmt.Errorf("POST predict: %v", err)
+		slog.Error("post predict", "error", err)
+		return errors.New("runner has crashed, check the log for additional detail")
```

Comment:
```suggestion
    return errors.New("model runner has unexpectedly stopped, this may be due to resource limitations or an internal error, check ollama server logs for details")
```

Here's my a wording suggestion, this error might be displayed in clients so I thought additional context might be good.